Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -133,7 +138,43 @@ extension Card {
         return [3]
       }
     }
-
+      
+    public var rawValue: String {
+        switch self {
+        case .unknown:
+            return "unknown"
+        case .mada:
+            return "mada"
+        case .visa:
+            return "visa"
+        case .mastercard:
+            return "mastercard"
+        case .maestro:
+            return "maestro"
+        case .americanExpress:
+            return "americanExpress"
+        case .discover:
+            return "discover"
+        case .dinersClub:
+            return "dinersClub"
+        case .jcb:
+            return "jcb"
+        }
+    }
```

Comment:
Do we need string raw value?

Reply:
Yes, all existing behaviour relies on that, for decoding from network, encoding and parsing.